src/elements/content-picker/shareAccessCellRenderer.js

```diff
@@ -15,11 +16,24 @@ export default (
     extensionsWhitelist: string[],
     hasHitSelectionLimit: boolean,
 ) => ({ rowData }: { rowData: BoxItem }) => {
-    if (!isRowSelectable(selectableType, extensionsWhitelist, hasHitSelectionLimit, rowData)) {
+    const { permissions = {} }: BoxItem = rowData;
+    const { can_set_share_access: rowCanSetShareAccess }: BoxItemPermission = permissions;
```

Comment:
Should this be `itemCanSetShareAccess` to match the similar usage in ContentPicker? You could also consider using `lodash/get` to cut down on the multiple destructures/default values in both spots: 

```
const itemCanSetShareAccess = getProp(rowData, 'permissions.can_set_share_access', false);
```